pom.xml

```diff
@@ -677,6 +677,10 @@
             <version>${surefire.version}</version>
             <configuration combine.self="override">
               <systemProperties>
+                <property>
+                  <name>skipRunnerV2Test</name>
+                  <value>true</value>
+                </property>
```

Comment:
As discussed offline, I don't think we need this property, we can just key off of the `SkipRunnerV2Test` category for all suites